SourceKitStressTester/Sources/Common/Message.swift

```diff
@@ -554,6 +566,12 @@ extension SourceKitError: CustomStringConvertible {
       let prefix = String(source.utf8.prefix(upTo: index))!
       let suffix = String(source.utf8.suffix(from: index))!
       return prefix + "<complete-offset>" + suffix
+    case .codeCompleteClose(let document, let offset):
+      guard let source = document.modification?.content else { return nil }
+      let index = source.utf8.index(source.utf8.startIndex, offsetBy: offset)
+      let prefix = String(source.utf8.prefix(upTo: index))!
+      let suffix = String(source.utf8.suffix(from: index))!
+      return prefix + "<complete-offset>" + suffix
```

Comment:
We could extract this snippet into a separate function, it's duplicated quite a few times. "replace" could be optional for the ones that need it